adm/src/main.rs

```diff
@@ -107,9 +107,11 @@ fn parse_args<'a>() -> ArgMatches<'a> {
             (@arg quiet: -q --quiet "do not display output"))
         (@subcommand genesis =>
             (about: "creates the genesis.batch file for initializing the validator")
-         (@arg input_file:
-          +takes_value ... "file or files containing batches to add to the resulting")
-            (@arg output: -o --output "choose the output file for GenesisData"))
+            (@arg input_file:
+             +takes_value ... "file or files containing batches to add to the resulting")
+            (@arg output: -o --output "choose the output file for GenesisData")
+            (@arg ignore-required-settings: --ignore-required-settings
+             "skip checking for settings that are required at genesis"))
```

Comment:
Wording nitpick: "skip the check for required settings"
or
"skip the check for settings that are required at genesis"

Also, would be to add a parenthetical phrase explaining why you'd do this; for example:

"skip the check for required settings (intended for backward and forward compatibility)"

... or something like that.

Reply:
It's not about backward compatibility but for supporting alternative settings transaction families.  The Sawtooth settings family is a reference implementation.

Reply:
Correct. Backward compatibility would be a nice bonus.

Reply:
Updated